admin/configuration.php

```diff
@@ -21,7 +21,14 @@
           $messageStack->add_session(ERROR_ADMIN_DEMO, 'caution');
           zen_redirect(zen_href_link(FILENAME_CONFIGURATION, 'gID=' . $_GET['gID'] . '&cID=' . (int)$cID));
         }
+        
         $configuration_value = zen_db_prepare_input($_POST['configuration_value']);
+        // See if there are any configuration checks 
+        $checks = $db->Execute("SELECT val_function FROM " . TABLE_CONFIGURATION . " WHERE configuration_id = '" . (int)$cID . "'");
+       if (!$checks->EOF && $checks->fields['val_function'] != NULL) { 
+           require_once("includes/configuration_checks.php"); 
```

Comment:
Can we move the file into the `admin/includes/functions/` folder instead of just `admin/includes`?

And while in this section, touch the indenting of your `if` statement and the closing `}` to add one more space so they align with the lines above and below this section.